src/PJ_calcofi.c

```diff
@@ -174,8 +174,8 @@ int pj_calcofi_selftest (void) {
     double tolerance_lp = 1e-10;
     double tolerance_xy = 1e-7;
 
-    char e_args[] = {"+proj=calcofi   +ellps=GRS80  +lat_1=0.5 +lat_2=2"};
-    char s_args[] = {"+proj=calcofi   +a=6400000    +lat_1=0.5 +lat_2=2"};
+    char e_args[] = {"+proj=calcofi   +ellps=GRS80  +lat_1=0.5 +lat_2=2 +no_defs"};
```

Comment:
The pj_inv/fwd scaling "bug" was detected due to this projection stomping on setup data.